sdk/eventhub/mock-hub/src/services/eventHubs.ts

```diff
@@ -294,7 +301,7 @@ export class MockEventHub {
         // Probably should close the sender at this point.
         event.sender.close({
           condition: "amqp:internal-error",
-          description: err?.message ?? ""
+          description: (err as any)?.message ?? ""
```

Comment:
This is probably just defensive, I'm guessing. Do you ever get an error that doesn't have a message? Are there other fields that are useful on it?

Reply:
So the most recent version of TypeScript causes the type of `err` in the `catch` block to be `unknown` instead of `any`. VSCode was yelling at me because you can't access properties of unknown without first checking for their existence. This is just defensive at this point.

Reply:
Makes sense. 

Also, that seems like a good change in TypeScript. Just barely missed that update myself I suppose :)